### DISCUSSION TITLE
Intresting emulation bug.

### DISCUSSION THREAD

**Discussion by SandMartininbox** [86Box/86Box#4909] [General]
I created a virtual machine with Windows 98 SE, and installed the game Jane's IAF on it.
So, this is the only virtual machine (along with PCEM) that creates a bug - namely, the plane appears without fuel when taking off from the runway.
I tried the same thing in Virtual Box, it doesn't have such a bug, but the graphics are glitchy there.

**Comment by alanthegrate** [comment]
Can anyone help because the Emulator works so munch better for Jane's IAF. but I get the No fuel glitch with PCEM but it works so smooth what am I missing?

**Reply by jriwanek** [reply]
What build of 86box are you using? Old Dynarec, New dynarec? Details about your host and guest configurations? (Provide your 86box.cfg)

**Comment by Cacodemon345** [comment]
@SandMartininbox @alanthegrate Try build 7262 and see if this still happens.

**Comment by pankoza2-pl** [comment]
Also, try enabling Softfloat FPU
